test/modules/generic-mqtt-tester/src/message_channel.rs

```diff
@@ -58,26 +51,22 @@ impl MessageHandler for ReportResultMessageHandler {
         let sequence_number = parse_sequence_number(&received_publication);
         let batch_id = Uuid::from_u128_le(received_publication.payload.slice(4..20).get_u128_le());
 
-        if self.batch_id == batch_id {
-            info!(
-                "reporting result for publication with sequence number {}",
-                sequence_number,
-            );
-            let result = MessageTestResult::new(
-                self.tracking_id.clone(),
-                self.batch_id.to_string(),
-                sequence_number,
-            );
-
-            let test_type = trc_client::TestType::Messages;
-            let created_at = chrono::Utc::now();
-            self.reporting_client
-                .report_result(self.report_source.clone(), result, test_type, created_at)
-                .await
-                .map_err(MessageTesterError::ReportResult)?;
-        } else {
-            error!("received publication with non-matching batch id");
-        }
+        info!(
+            "reporting result for publication with sequence number {}",
+            sequence_number,
+        );
+        let result = MessageTestResult::new(
+            self.tracking_id.clone(),
+            batch_id.to_string(),
+            sequence_number,
+        );
+
+        let test_type = trc_client::TestType::Messages;
+        let created_at = chrono::Utc::now();
+        self.reporting_client
+            .report_result(self.report_source.clone(), result, test_type, created_at)
+            .await
+            .map_err(MessageTesterError::ReportResult)?;
```

Comment:
What was the motivation behind removing the batchId?

Reply:
https://github.com/Azure/iotedge/pull/4883#discussion_r618814887

Reply:
Makes sense!